File: two_sum.go
func twoSum(nums []int, target int) []int {
    for x:=0; x<len(nums); x++ {
        for y:=x+1; y<len(nums); y++ {
            if nums[x]+nums[y] == target {
                r := []int{x, y};
                return r;
            }
        }
    }
    return nums;
}

File: two_sum.swift
class Solution {
    func twoSum(_ nums: [Int], _ target: Int) -> [Int] {
        for num in (0..<nums.count) {
            for num2 in ((num+1)..<nums.count) {
                if (nums[num]+nums[num2]) == target {
                    return [num, num2]
                }
            }
        }
        return nums
    }
}
